### ISSUE TITLE
Largest Rectange Area (Histogram)

### ISSUE THREAD

**Issue by sudipg4112001** [Algo-Phantoms/Algo-Tree#161]
I would like to code using stack for finding the largest area available. Here, I would input a set of heights of a building and a  number of buildings available.Then, find the largest area available or vacant.
I am a participant of GSSOC'21

**Comment by krishna-NIT** [comment]
I am a GSSOC '21 participant.

I wish to contribute a JAVA code for this issue:

My code will have comment lines to make it explanatory and understandable to everyone.
I will be very happy if I am assigned this issue and allowed to work on this.

**Comment by sudipg4112001** [comment]
Yeah, we can work together same thing demonstrated in two different languages sounds good, because I would preferably do in C++ or python

**Comment by sudipg4112001** [comment]
@plazzy99 @rudrakshi99 please assigned. If you like the idea

**Comment by Vansh-Gupta4** [comment]
i like to contribute to this project ...my code will be in java with proper comments and test cases

**Comment by kshittijagrawal** [comment]
Hey there!
I'd like to contribute to the project. I want to address and fix this issue. My implementation will be in python with a clean code and documentation. Kindly provide me with this issue.
I'm also a GSSOC'21 participant.

**Comment by plazzy99** [comment]
Mention the algorithm you are gonna going to apply @sudipg4112001

**Comment by plazzy99** [comment]
> Yeah, we can work together same thing demonstrated in two different languages sounds good, because I would preferably do in C++ or python

Go with python

**Comment by sudipg4112001** [comment]
Should I only upload the code, or something additional to it ?

**Comment by einhawking** [comment]
I am a participant of GSSOC'21
I would like to contribute in this

**Comment by tarun26091999** [comment]
@einhawking language??
